monkeys/schema.json

```diff
@@ -198,6 +198,11 @@
                         "description": "How many users should be created in this team",
                         "type": "integer"
                     },
+                    "2FAEnabled": {
+                        "description": "Does this server require 2FA authentication?",
+                        "type": "boolean",
+                        "default": false
+                    },
```

Comment:
Not sure if relevant for Monkeys, but it may also depend on each team. Some teams may have it, some may not, in the same server.

Why store this, if this can be figured out during runtime, based on the login response?

Reply:
We need it to create the users as well. When we're setting the user handle we need to login. And since I use the backdoor to get the code I don't actually need to make a failed attempt to login to get the code.

Reply:
And the config is per team. So we can have multiple teams on the same server with different configs.

Reply:
Ahh cool.
Also, just to correct myself. It seems it's not per team, but indeed _per backend_  for some technical/business reason?
In any case, it makes sense for Monkeys!

Reply:
This is inheritance from the first iteration of the monkeys 😅

I should have renamed it. It is used to represent the team and its backend configs.